packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -607,6 +616,42 @@
 
   </aggregation>
 
+  <aggregate_qualities>
+
+    <parameter>
+      <name>aggregate qualities: check symmetry</name>
+      <type>bool</type>
+      <default>false</default>
+      <description>Whether to check symmetry and use nonsymmetric aggregate quality estimate if necessary.</description>
+      <comment-ML>parameter not existing in ML</comment-ML>
+    </parameter>
+
+    <parameter>
+      <name>aggregate qualities: good aggregate threshold</name>
+      <type>double</type>
+      <default>100.0</default>
+      <description>Maximum quality where aggregate is considered acceptable</description>
+      <comment-ML>parameter not existing in ML</comment-ML>
+    </parameter>
+
+    <parameter>
+      <name>aggregate qualities: file output</name>
+      <type>bool</type>
+      <default>false</default>
+      <description>Maximum quality where aggregate is considered acceptable</description>
```

Comment:
Description seems to be wrong.

Reply:
Thanks for the catch. Made exactly the mistake you suggested. Should be fixed now